src/theme/sass/theme/_page.scss

```diff
@@ -19,7 +19,7 @@
 }
 
 body {
-  background: $grey;
+  background: $white;
```

Comment:
This rule is actually broken (but working anyways). Before this code:
```
body {
  background: $grey;
}
```
was using the `$` in front of grey here to reference a [sass variable](https://sass-lang.com/documentation/variables) to pull in a shared value defined in [`src/global/sass/foundations/_colors.scss`](https://github.com/badgerherald/badgerherald.com/blob/master/src/global/sass/foundations/_colors.scss). The variable `$white` isn't actually defined anywhere, so this css rule is actually broken—but working regardless (since the default is white).

tl;dr—drop the `$` in front of `$white` to reference the HTML color value `white`